Jenkinsfile

```diff
@@ -1158,13 +1158,156 @@ pipeline {
+              enc_dec_nmt-bottleneck.py \
+              --config-path=conf \
+              --config-name=aayn_bottleneck \
+              do_testing=true \
+              model.model_type=nll \
+              model.encoder.arch=seq2seq \
+              model.encoder.hidden_steps=1 \
+              model.encoder.hidden_blocks=1 \
+              model.encoder.hidden_init_method=params \
+              model.train_ds.src_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-en-de.src \
+              model.train_ds.tgt_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-en-de.ref \
+              model.validation_ds.src_file_name=[/home/TestData/nlp/nmt/toy_data/wmt13-en-de.src,/home/TestData/nlp/nmt/toy_data/wmt14-en-de.src] \
+              model.validation_ds.tgt_file_name=[/home/TestData/nlp/nmt/toy_data/wmt13-en-de.ref,/home/TestData/nlp/nmt/toy_data/wmt14-en-de.ref] \
+              model.test_ds.src_file_name=/home/TestData/nlp/nmt/toy_data/wmt13-en-de.src \
+              model.test_ds.tgt_file_name=/home/TestData/nlp/nmt/toy_data/wmt13-en-de.ref \
+              model.encoder_tokenizer.tokenizer_model=/home/TestData/nlp/nmt/toy_data/tt_tokenizer.BPE.4096.model \
+              model.decoder_tokenizer.tokenizer_model=/home/TestData/nlp/nmt/toy_data/tt_tokenizer.BPE.4096.model \
+              trainer.gpus=[0] \
+              +trainer.fast_dev_run=true \
+              +trainer.limit_test_batches=2 \
+              exp_manager=null \
+              '
+            }
+        }
+        stage('L2: NMT Perceiver Encoder Bridge (identity init)') {
+            steps {
+              sh 'cd examples/nlp/machine_translation && \
+              enc_dec_nmt-bottleneck.py \
+              --config-path=conf \
+              --config-name=aayn_bottleneck \
+              do_testing=true \
+              model.model_type=nll \
+              model.encoder.arch=bridge \
+              model.encoder.hidden_steps=1 \
+              model.encoder.hidden_blocks=1 \
+              model.encoder.hidden_init_method=identity \
+              model.train_ds.src_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-en-de.src \
+              model.train_ds.tgt_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-en-de.ref \
+              model.validation_ds.src_file_name=[/home/TestData/nlp/nmt/toy_data/wmt13-en-de.src,/home/TestData/nlp/nmt/toy_data/wmt14-en-de.src] \
+              model.validation_ds.tgt_file_name=[/home/TestData/nlp/nmt/toy_data/wmt13-en-de.ref,/home/TestData/nlp/nmt/toy_data/wmt14-en-de.ref] \
+              model.test_ds.src_file_name=/home/TestData/nlp/nmt/toy_data/wmt13-en-de.src \
+              model.test_ds.tgt_file_name=/home/TestData/nlp/nmt/toy_data/wmt13-en-de.ref \
+              model.encoder_tokenizer.tokenizer_model=/home/TestData/nlp/nmt/toy_data/tt_tokenizer.BPE.4096.model \
+              model.decoder_tokenizer.tokenizer_model=/home/TestData/nlp/nmt/toy_data/tt_tokenizer.BPE.4096.model \
+              trainer.gpus=[0] \
```

Comment:
I think we can only really run two tests in parallel since there are only two gpus on some of the CI machines. so you can set one test to `trainer.gpus=[0]` and the other test to `trainer.gpus=[1]`

Reply:
Done.